package/yast2-storage-ng.spec

```diff
@@ -49,6 +49,8 @@ BuildRequires:  rubygem(%{rb_default_ruby_abi}:parallel_tests)
 Requires:       findutils
 # Encryption#use_key_file_in_commit
 Requires:       libstorage-ng-ruby >= 4.5.144
```

Comment:
Isn't it redundant now when it requires the exact version anyway?

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1235598#c8